babel.config.js

```diff
@@ -12,6 +13,38 @@ const targets = isBrowser
 	? { browsers: [ 'last 2 versions', 'Safari >= 10', 'iOS >= 10', 'ie >= 11' ] }
 	: { node: 'current' };
 
+const extensionOverrides = [
+	[
+		'@wordpress/import-jsx-pragma',
+		{
+			scopeVariable: 'createElement',
+			source: '@wordpress/element',
+			isDefault: false,
+		},
+	],
+	[
+		'@babel/transform-react-jsx',
+		{
+			pragma: 'createElement',
+		},
+	],
+];
+
+// The output directory is set when SDK runs, so we only set the POT generator override in that case.
+if ( outputDir ) {
+	extensionOverrides.unshift( [
+		'@wordpress/babel-plugin-makepot',
+		{
+			output: outputDir + '/extensions.pot',
```

Comment:
Let's use `path` utility to combine these to ensure it works well in every OS and even if `outputDir` contains trailing slash or not:

```js
path.join( outputDir, 'extensions.pot' )
```

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/30303/commits/fe2733904a9e0e5d954bc2ee2245c5a888f94e9d